requirements.txt

```diff
@@ -4,3 +4,7 @@ flake8==3.5.0
 pexpect==4.2.1
 pycrypto==2.6.1
 pyserial==3.4
+ed25519==1.4
+cbor==1.0.0
+cryptography==2.6.1
+click==7.0
```

Comment:
This can be removed.

Reply:
@kaspar030 seems like an easy fix ;)
so let's merge this PR the last one preliminary to #11818 ?

Reply:
done